src/dscanner/main.d

```diff
@@ -37,12 +37,14 @@ import inifiled;
 
 import dsymbol.modulecache;
 
+@safe:
+
 version (unittest)
-	void main()
+void main() @trusted
 {
 }
 else
-	int main(string[] args)
+int main(string[] args) @trusted
```

Comment:
What is the point of this change?

Reply:
shouldn't main be `@safe` instead of `@trusted`?